sdk/core/src/context.rs

```diff
@@ -0,0 +1,20 @@
+/// Pipeline execution context.
+///
+/// During a pipeline execution, context will be passed from the function starting the
+/// pipeline down to each pipeline policy. Contrarily to the Request, the context can be mutated
+/// by each pipeline policy and is not reset between retries. It can be used to pass the whole
+/// pipeline execution history between policies.
+/// For example, it could be used to signal that an execution failed because a CosmosDB endpoint is
+/// down and the appropriate policy should try the next one).
+#[derive(Clone)]
+pub struct Context {
+    // Temporary hack to make sure that Context is not initializeable
```

Comment:
Nit: use TODO or BUGBUG to find these easily and fix them before release.

Reply:
Good idea. 👍